core/src/com/unciv/logic/civilization/CivilizationInfo.kt

```diff
@@ -436,7 +436,7 @@ class CivilizationInfo {
         policies.endTurn(nextTurnStats.culture.toInt())
 
         // disband units until there are none left OR the gold values are normal
-        if (!isBarbarian() && gold < -100 && nextTurnStats.gold.toInt() < 0) {
+        if (!isBarbarian() && !gameInfo.gameParameters.godMode && gold < -100 && nextTurnStats.gold.toInt() < 0) {
```

Comment:
These extra units would just disband anyway on the first turn of the player, not sure what we gain here

Reply:
Scenario creators are not annoyed while testing something. 
But yes will disband if scenario creator did not check gold before releasing scenario.

Reply:
well maybe having units deleted will help them not to forget to check and edit in save gold amount